tests/bulk_test.py

```diff
@@ -206,6 +206,7 @@ def test_bulk_save_pandas(self, mock_cursor):
         # Verify number of batches.
         self.assertEqual(effective_op_count, OPCOUNT)
 
+    @skipIf(sys.version_info >= (3, 13), "SQLAlchemy/Dask is not supported on Python >=3.13 yet")
```

Comment:
That's because pyarrow, a dependency of dask-expr, does not provide binary wheels for Python 3.13 yet.

- https://github.com/apache/arrow/issues/44443